media-video/mpv/mpv-0.27.2.recipe

```diff
@@ -17,9 +17,21 @@ CHECKSUM_SHA256_2="66f3983fcc6dc0d17393514b2b52063702004800428327db3597754dd05c3
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ -n "$secondaryArchSuffix" ]; then
+# Comment out this block if you wish to co-install zstd_bin & e.g. zstd_x86_bin.
```

Comment:
```diff
-# Comment out this block if you wish to co-install zstd_bin & e.g. zstd_x86_bin.
```
mpv != zstd :smile:

Reply:
There is a well known italian dish called "Copy Pasta".